### PR TITLE
Fix iife build

### LINKED ISSUE #1553: IIFE build is broken
### Version
1.0.0

### Reproduction link
[https://jsfiddle.net/chrisvfritz/50wL7mdz/](https://jsfiddle.net/chrisvfritz/50wL7mdz/)

### Steps to reproduce
Open the following HTML in the browser and observe console error message

```
<html>
  <body>
    Open the console to observe error message
    <script>
      // hide warning message in karma test
      // see https://github.com/vuejs/vue-test-utils/issues/42
      var VueTemplateCompiler = {}
    </script>
    <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
    <script src="https://unpkg.com/@vue/test-utils@1.0.0/dist/vue-test-utils.iife.js"></script>
  </body>
</html>
```

### What is expected?
IIFE should load without error

### What is actually happening?
This new IIFE fails to load in the browser with "Uncaught ReferenceError: testUtils is not defined at vue-test-utils.iife.js:13932"

---
Last working IIFE was in 1.0.0-beta.33.  Broken by #1518 (Optionally hide deprecation errors) which introduces a new import "testUtils" to packages/shared/utils.js.  When this new import gets compiled into node_modules/@vue/test-utils/dist/vue-test-utils.iife.js it changes the IIFE signature from `(function (...) {...}({}, Vue, VueTemplateCompiler));` to `(function (...) {...}({}, Vue, VueTemplateCompiler, testUtils));`.

### PR DESCRIPTION
**Pull request by lmiller1990**
resolves #1553 , hopefully.

### CODE REVIEW

File: packages/shared/util.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import Vue from 'vue'
 import semver from 'semver'
 import { VUE_VERSION } from './consts'
-import { config } from '@vue/test-utils'
+import { config } from '../test-utils/src'
```
Comment (lmiller1990) [inline packages/shared/util.js:5]: not sure it's ideal to import across packages but 🤷 seems to work great here.

I don't know why this project is a mono-repo.

**Comment by afontcu** [comment]
> hopefully

Is there any way to test it out?

**Comment by lmiller1990** [comment]
@afontcu I tested it in my browser by creating an `index.html` and having Vue globally available, and using the iife build - works fine.

I hope it doesn't break anything else. All tests are green... but they were before too. We probably need some way to test the iife build, but it's nothing something I can put time into right now. I guess we need to set up selenium with nightwatch, or something.

**Comment by afontcu** [comment]
> @afontcu I tested it in my browser by creating an `index.html` and having Vue globally available, and using the iife build - works fine.

"it works on my machine!!" 😂 

joking aside, build is already broken so as long as tests are green… 👍

**Review by afontcu — APPROVED** [review]

**Comment by sullivanpt** [comment]
>  We probably need some way to test the iife build, but it's nothing something I can put time into right now. I guess we need to set up selenium with nightwatch, or something.

Actually we don't need full blown selenium. We can test the iife in node with karma + puppeteer -- we use karma + phantomjs but I don't recommend it because it's not maintained and going stale. 

~~Anyway, if I have free time I'll try to pull this together.~~ (nvm. the yarn+ts+flow learning curve is a bit much for me to contribute directly. sorry)

Thanks so much for fixing the IIFE so quickly!

**Comment by lmiller1990** [comment]
I will release early next week @sullivanpt !

If you need it now, clone the repo, `yarn install && yarn build` and grab the iife build from `dist`.

**Pull request merged**